content/integrations/aws-codebuild.mdx

```diff
@@ -171,6 +171,25 @@ phases:
       - depot build -t <your-ecr-registry-url>:latest --push .
 ```
 
+#### Obtaining an authenticated Docker config.json
+
+Any easy way to obtain a pre-authenticated `config.json` is to login to the Docker registry and copy the `config.json` file. We can use Docker again to authenticate and grab the config file without modifying our local Docker configuration.
+
+```bash
```

Comment:
I think we should cut this part about using a Docker container to do this. Just going straight to the `docker login` stuff is sufficient.